src/components/keyboard-shortcut-controls.tsx

```diff
@@ -0,0 +1,93 @@
+import React, { FormEvent, useState } from "react";
+
+import "./keyboard-shortcut-controls.scss";
+
+interface IProps {
+  shortcutEnabled: boolean;
+  shortcutKeys?: string;
+  onCustomizeShortcut?: (shortcut: string) => void;
+  onToggleShortcut: () => void;
+}
+
+export const KeyboardShortcutControls = (props: IProps) => {
+  const { shortcutEnabled, shortcutKeys, onCustomizeShortcut, onToggleShortcut } = props;
+  const toggleButtonLabel = shortcutEnabled ? "Disable Shortcut" : "Enable Shortcut";
+  const [showError, setShowError] = useState(false);
+  const [showConfirmation, setShowConfirmation] = useState(false);
+
+  const handleToggleShortcut = () => {
+    onToggleShortcut();
+  };
+
+  const handleCustomizeShortcut = (event: FormEvent) => {
+    event.preventDefault();
+    const form = event.target as HTMLInputElement;
+    const shortcut = form.querySelector("input")?.value;
+    if (shortcut) {
+      onCustomizeShortcut?.(shortcut);
```

Comment:
You should probably do a `trim` on the input.  I thought I broke it when I just had a space after the shortcut.